pkgs/top-level/python-packages.nix

```diff
@@ -23,6 +23,8 @@ let
 
   buildPythonApplication = args: buildPythonPackage ({namePrefix="";} // args );
 
+  pypi-sources = builtins.fromJSON (builtins.readFile ../development/python-modules/pypi-sources.json);
```

Comment:
you can use the shortcut `pypi-sources = lib.importJSON ../development/python-modules/pypi-sources.json`